packages/doenetml-worker-rust/lib-doenetml-core/src/components/doenet/division.rs

```diff
@@ -49,19 +50,35 @@ mod component {
            )]
         CodeNumber,
 
-        /// How many levels deep this `<section /> is nested.
+        /// How many levels deep this `<division /> is nested.
         #[prop(
                value_type = PropValueType::Integer,
                profile = PropProfile::DivisionDepth,
                for_render
            )]
         DivisionDepth,
 
-        /// The label that should be used to refer to this `<section>` when
-        /// the section is referred to by `<xref>`.
+        /// The type of this `<division />`. E.g. "section", "chapter", "subsection", etc.
+        #[prop(
+               value_type = PropValueType::DivisionType,
+               profile = PropProfile::DivisionType,
+               for_render
+           )]
+        DivisionType,
+
+        /// The value of the `division` attribute.
```

Comment:
If we hadn't had a previous conversation about `FooAttr` props for temporary processing, I would have been confused by this. I suggest adding to the docstring of `DivisionTypeAttr` that this is an internal intermediate value used to calculate the `DivisionType` prop.